airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/JobPersistence.java

```diff
@@ -162,15 +162,27 @@
   /// ARCHIVE
 
   /**
-   * Returns the AirbyteVersion stored in the database
+   * Returns the AirbyteVersion.
    */
   Optional<String> getVersion() throws IOException;
 
   /**
-   * Set the database to @param AirbyteVersion
+   * Set the airbyte version
    */
   void setVersion(String airbyteVersion) throws IOException;
 
+  /**
+   * Returns a deployment UUID.
+   */
+  Optional<UUID> getDeployment() throws IOException;
```

Comment:
leaving the deployment id -> all workspaces for a deployment comment here is also fine.